### PR TITLE
DCMAW-9802: Rename variable and method names for consistency and readability

### PR DESCRIPTION
**Pull request by J-son1**
​DCMAW-9802
### What changed
Rename declaration of variables and service method names to be consistent and readable in the /async/token and /async/credential Lambdas.

## Checklists

- [x] There is a ticket raised for this PR that is present in the branch name
- [x] No PII data logged. [See guidance here](https://govukverify.atlassian.net/wiki/spaces/DCMAW/pages/3502407722/PII+Logging+Considerations)
- [ ] Demo to a BA, TA, and the team.
- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

File: backend-api/src/functions/asyncCredential/asyncCredentialHandler.test.ts [hunk 1]
```diff
@@ -236,7 +236,7 @@ describe("Async Credential", () => {
         });
 
         dependencies.tokenService = () =>
-          new MockTokenServiceGetDecodedTokenFailure();
+          new MockTokenServiceGetDecodedTokenErrorResult();
```
Comment (sandy-may) [inline backend-api/src/functions/asyncCredential/asyncCredentialHandler.test.ts:239]: We're using "*Success" in our naming of passing mocks. Is there are a reason why we're using a different naming convention for the failing mocks?

File: backend-api/src/functions/asyncCredential/asyncCredentialHandler.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ import {
   IDecodedToken,
   IDecodeToken,
   IVerifyTokenSignature,
```
Comment (sandy-may) [inline backend-api/src/functions/asyncCredential/asyncCredentialHandler.ts:11]: thank you for this :)

File: backend-api/src/functions/asyncToken/asyncTokenHandler.ts [hunk 3]
```diff
@@ -33,101 +33,103 @@ export async function lambdaHandlerConstructor(
...
-  const suppliedCredentials = processRequest.value;
+  const suppliedClientCredentials = processRequestResult.value;
 
   // Fetching stored client credentials
-  const clientCredentialsService = dependencies.clientCredentialService();
-  const clientCredentialsResult =
-    await clientCredentialsService.getClientCredentials();
-  if (clientCredentialsResult.isError) {
+  const clientCredentialsService = dependencies.clientCredentialsService();
+  const storedClientCredentialsArrayResult =
```
Comment (sandy-may) [inline backend-api/src/functions/asyncToken/asyncTokenHandler.ts:67]: I don't think we need Array in the name. I'm also not sure why we've introduced "stored"

File: backend-api/src/functions/services/clientCredentialsService/clientCredentialsService.test.ts [hunk 4]
```diff
@@ -210,11 +212,11 @@ describe("Client Credentials Service", () => {
           });
 
           clientCredentialsService.resetCache();
-          await clientCredentialsService.getClientCredentials();
+          await clientCredentialsService.getStoredClientCredentials();
```
Comment (sandy-may) [inline backend-api/src/functions/services/clientCredentialsService/clientCredentialsService.test.ts:215]: I think this should be getRegisteredClientCredentials to align with our domain naming?

File: backend-api/src/functions/asyncCredential/asyncCredentialHandler.test.ts [hunk 5]
```diff
@@ -999,7 +999,7 @@ class MockFailingClientCredentialsServiceGetClientCredentialsById
   validateAsyncCredentialRequest(): Result<null> {
     return successResult(null);
   }
-  getClientCredentialsById(): Result<IClientCredentials> {
+  getStoredClientCredentialsById(): Result<IClientCredentials> {
```
Comment (sandy-may) [inline backend-api/src/functions/asyncCredential/asyncCredentialHandler.test.ts:1002]: getRegisteredClientCredentialsById

File: backend-api/src/functions/services/clientCredentialsService/clientCredentialsService.ts [hunk 6]
```diff
@@ -24,7 +24,9 @@ export class ClientCredentialsService
     this.cacheTTL = 3600 * 1000;
   }
 
-  getClientCredentials = async (): Promise<Result<IClientCredentials[]>> => {
+  getStoredClientCredentials = async (): Promise<
```
Comment (sandy-may) [inline backend-api/src/functions/services/clientCredentialsService/clientCredentialsService.ts:27]: getRegisteredClientCredentials

File: backend-api/src/functions/services/clientCredentialsService/clientCredentialsService.ts [hunk 7]
```diff
@@ -84,13 +86,13 @@ export class ClientCredentialsService
       return errorResult("Client secret not valid for the supplied clientId");
     }
 
-    const registeredRedirectUri = storedCredentials.redirect_uri;
-    if (!registeredRedirectUri) {
+    const storedRedirectUri = storedCredentials.redirect_uri;
```
Comment (sandy-may) [inline backend-api/src/functions/services/clientCredentialsService/clientCredentialsService.ts:89]: registeredRedirectUri

File: backend-api/src/functions/services/clientCredentialsService/clientCredentialsService.ts [hunk 8]
```diff
@@ -101,7 +103,7 @@ export class ClientCredentialsService
   validateAsyncCredentialRequest = (
     config: IValidateAsyncCredentialRequestConfig,
   ): Result<null> => {
-    const registeredRedirectUri = config.storedCredentials.redirect_uri;
+    const registeredRedirectUri = config.storedClientCredentials.redirect_uri;
```
Comment (sandy-may) [inline backend-api/src/functions/services/clientCredentialsService/clientCredentialsService.ts:106]: registeredClientCredentials

I don't think 'config' is the right term for this input?

File: backend-api/src/functions/asyncCredential/asyncCredentialHandler.test.ts (hunk 1 above)
Comment (J-son1) [reply backend-api/src/functions/asyncCredential/asyncCredentialHandler.test.ts:239]: I agree with using SuccessResult. I'll update those ones

Comment (J-son1) [reply backend-api/src/functions/asyncCredential/asyncCredentialHandler.test.ts:240]: Updated

**Review by sandy-may — APPROVED** [review]

**Pull request merged**